.changeset/kind-rings-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-cell": minor
```

Comment:
Do we want to include the tokens package too? (might be fine without too if you have other PRs in the feature branch that will mark the `tokens` package!)

Reply:
yep, thanks for the reminder! the `patch` will go in the next PR. btw Do you think it should be `patch` or `minor`?

Reply:
Hmmm good question. I was thinking `minor` because it adds new css variables.. though the css variables aren't really a part of the public api right now (for example, we don't include the component css variables in the docs)! So maybe a `patch`!

If we have a separate package for the css variables for all packages, I think that would make more sense for that to be a minor update!